### ISSUE TITLE
ActionView::Template::Error (wrong number of arguments (given 2, expected 0..1)

### ISSUE THREAD

**Issue by TablGithub** [livingstyleguide/livingstyleguide#234]
Getting ActionView::Template::Error (wrong number of arguments (given 2, expected 0..1) after bundle update

I ran bundle update in my project and it updated several gems. once I tried to run 'rails s' and open the site in localhost it gave me this error.

"Showing ../_head.html.erb where line #12 raised:
wrong number of arguments (given 2, expected 0..1)

10. <%= metamagic site: "Title", title: -> { title.include?(site) ? title : "#{title} - #{site}" if title} %>
11. <%= include_gon %>
12. <%= favicon_link_tag 'favicon.ico' %>
13. <%= csrf_meta_tags %>
14. <%= stylesheet_link_tag 'application', media: 'all' %>"

My system configurations are<
rbenv 1.1.2-2-g4e92322
ruby 2.5.1p57
Bundler version 1.17.2
Rails 5.2.2

I have noticed that livingstyleguide is updated to 2.1.0. Once I have build the application without the livingstyleguide gem, all seems fine. And I when I have updated my Gemfile to install 2.0.3 version, again all seems fine. Looks like a issue with using 2.1.0.

**Comment by hagenburger** [comment]
Hey @TablGithub, as the switch from `sass` to `sassc` is the only change between 2.0.3 and 2.1.0, it’s pretty hard to get an idea what could probably cause this. 

If you comment out line 12, does a similar error happen elsewhere?

**Comment by TablGithub** [comment]
Yes same exceptions throws at line number 14 again. using 2.0.3 is the only solution worked for me.

**Comment by hagenburger** [comment]
This seems very tricky. I’ll be open for specific questions whenever I can help if you investigate on this.

**Comment by rmehner** [comment]
Btw. this is the error message I'm seeing with #233, when I try to swap in sassc-rails for sass-rails, so that might be related.

Just adding more information, still need to figure out what the root cause is :)

**Comment by andyduong1920** [comment]
But `2.0.3` doesn't work with the new sassc 😢
